src/Util.ts

```diff
@@ -45,7 +45,7 @@ export function options(
     urlOrQueryOrSearch: URL | URLSearchParams | string | object | undefined = typeof location === 'undefined'
         ? undefined
         : location
-): object {
+): any {
```

Comment:
and yes, I regret, I am fully agree with GPT, and also when he says that there is rare case where it's appropriat. It's the case here otherwise you can't use the function in TS. If we look deeply these functions `Util.options` or `Util.objectFrom` are similar to `Object.assign` (declared with any as well): it's generic function to parse generic object. I tried `unknown `or `object` and it doesn't allow to use the function
```
const options = Util.options();
if(options.nameQuery) { .... } // Doesn't build
```

Reply:
Strange, very strange